_pytest/mark.py

```diff
@@ -67,10 +67,6 @@ def extract_from(cls, parameterset, legacy_force_tuple=False):
 
         return cls(argval, marks=newmarks, id=None)
 
-    @property
-    def deprecated_arg_dict(self):
```

Comment:
I suppose this was never part of the public API given that it is a method of the new `ParameterSet` class, but if I'm wrong then we should add a `removal` changelog note.